### PR TITLE
Show textures created with `gui.new_texture()` in resource profiler.

### LINKED ISSUE #7742: Resource profiler: extend output with information about texture memory usage
**Is your feature request related to a problem? Please describe (REQUIRED):**
The big project needs to know information about used texture memory per texture asset for:
- finding possibilities for optimization and define priority of them
- preparing early feedback about performance in content creation processes
- controlling releases by performance QA in the scope of video memory usage

**Describe the solution you'd like (REQUIRED):**
The output of resource profiler should include information about used texture memory per texture asset

**Describe alternatives you've considered (REQUIRED):**
The texture memory usage may be calculated manually / half-automatically from the information of texture resolution and applied texture compressions rules. But this way is not so scalable to be widely used in performance QA processes

### PR DESCRIPTION
**Pull request by AGulev**
Now textures created with `gui.new_texture()` will be listed in resource profiler altogether with the GUI resource name that owns it.

image

Fix https://github.com/defold/defold/issues/7742

## PR checklist

* [ ] Code
	* [ ] Add engine and/or editor unit tests.
	* [x] New and changed code follows the overall code style of existing code
	* [x] Add comments where needed
* [ ] Documentation
	* [ ] Make sure that API documentation is updated in code comments
	* [ ] Make sure that manuals are updated (in github.com/defold/doc)
* [ ] Prepare pull request and affected issue for automatic release notes generator
	* [x] Pull request - Write a message that explains what this pull request does. What was the problem? How was it solved? What are the changes to APIs or the new APIs introduced? This message will be used in the generated release notes. Make sure it is well written and understandable for a user of Defold.
	* [x] Pull request - Write a pull request title that in a sentence summarises what the pull request does. Do not include "Issue-1234 ..." in the title. This text will be used in the generated release notes.
	* [x] Pull request - Link the pull request to the issue(s) it is closing. Use on of the [approved closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
	* [x] Affected issue - Assign the issue to a project. Do not assign the pull request to a project if there is an issue which the pull request closes.
	* [ ] Affected issue - Assign the "breaking change" label to the issue if introducing a breaking change.
	* [ ] Affected issue - Assign the "skip release notes" is the issue should not be included in the generated release notes.

----------

Example of a well written PR description:

1. Start with the user facing changes. This will end up in the release notes.
1. Add one of the GitHub approved closing keywords
1. Optionally also add the technical changes made. This is information that might help the reviewer. It will not show up in the release notes. Technical changes are identified by a line starting with one of these:
   1. `### Technical changes` 
   1. `Technical changes:`
   2. `Technical notes:`

```
There was a anomaly in the carbon chroniton propeller, introduced in version 8.10.2. This fix will make sure to reset the phaser collector on application startup.

Fixes #1234

### Technical changes
* Pay special attention to line 23 of phaser_collector.clj as it contains some interesting optimizations
* The propeller code was not taking into account a negative phase.
```

### CODE REVIEW

File: editor/src/clj/editor/command_requests.clj [hunk 1]
```diff
@@ -91,6 +91,10 @@
    {:ui-handler :engine-profile-show
     :help "Open the Engine Profiler in a web browser."}
 
+   :engine-resource-profiler
+   {:ui-handler :engine-profile-show
+    :help "Open the Engine Resource Profiler in a web browser."}
+
```
Comment (matgis) [inline editor/src/clj/editor/command_requests.clj:97]: Shouldn't the `:ui-handler` be `:engine-resource-profile-show` here?

File: editor/src/clj/editor/engine_profiler.clj [hunk 2]
```diff
@@ -39,6 +39,13 @@
          (ui/open-url (format "%s/engine-profiler" (http-server/local-url web-server)))
          (ui/open-url (format "%s/engine-profiler?addr=%s:%d" (http-server/local-url web-server) (:address (targets/selected-target prefs)) port)))))
 
+(handler/defhandler :engine-resource-profile-show :global
+  (enabled? [prefs]
+       (some? (targets/selected-target prefs)))
+     (run [prefs]
+       (let [address (:address (targets/selected-target prefs))]
+         (ui/open-url (format "http://%s:8002/" address)))))
+
```
Comment (matgis) [inline editor/src/clj/editor/engine_profiler.clj:48]: Style: Fix indentation.
```suggestion
(handler/defhandler :engine-resource-profile-show :global
  (enabled? [prefs] (some? (targets/selected-target prefs)))
  (run [prefs]
       (let [address (:address (targets/selected-target prefs))]
         (ui/open-url (format "http://%s:8002/" address)))))

```

**Review by matgis — CHANGES_REQUESTED** [review]

File: editor/src/clj/editor/command_requests.clj (hunk 1 above)
Comment (AGulev) [reply editor/src/clj/editor/command_requests.clj:97]: thx

File: editor/src/clj/editor/engine_profiler.clj (hunk 2 above)
Comment (AGulev) [reply editor/src/clj/editor/engine_profiler.clj]: thx. fixed.

**Review by JCash — DISMISSED** [review]

**Review by matgis — APPROVED** [review]

**Pull request merged**